WebUI/lims/shared.py

```diff
@@ -127,3 +126,49 @@ def pagePrev(pageNum):
 
     # Just navigage back one
     return pageNum - 1
+
+def blankSet(rowsList):
+    for row in range(0,len(rowsList)):
+        for column in range(0, len(rowsList[row])):
+            if not rowsList[row][column]:
+                rowsList[row][column] =' '
+    return rowsList
```

Comment:
I think this could be done a bit easier:

```Python
size = len(rowsList)
bucket = [[' '] * size for ndx in range(size)]
```

Reply:
Actually, we do not need every element become blank.